MinPlatformPkg/Test/Library/TestPointCheckLib/DxeCheckPci.c

```diff
@@ -256,7 +265,7 @@ TestPointCheckPciResource (
   UINT16                            MinBus;
   UINT16                            MaxBus;
   BOOLEAN                           IsEnd;
-  
+
```

Comment:
Unnecessary changes?

Reply:
It is just whitespace removal which should not have been allowed in the file in the first place. My editor is configured to automatically remove extra whitespace. I can undo it but it is also removed in the upstream patch.

Reply:
Okay removed? Or add it back?